docs/source/content/deep_learning/multilayer.rst

```diff
@@ -0,0 +1,106 @@
+##########
+Multi-layer Perceptron
+##########
+
+.. contents::
+  :local:
+  :depth: 3
+
+********
+Overview
+********
+
+A multilayer perceptron (MLP) is a deep, artificial *neural network*. 
```

Comment:
I would bold neural network and a lot of the other key words. That way these words stand out from other italic words like activate and all in What is a node Section